src/ca65/instr.c

```diff
@@ -1031,9 +1031,9 @@ const InsTable* InsTab = (const InsTable*) &InsTab6502;
 */
 static unsigned char EATab[12][AM65I_COUNT] = {
     {   /* Table 0 */
-        0x00, 0x00, 0x05, 0x0D, 0x0F, 0x15, 0x1D, 0x1F,
-        0x00, 0x19, 0x12, 0x00, 0x07, 0x11, 0x17, 0x01,
-        0x00, 0x00, 0x00, 0x03, 0x13, 0x09, 0x00, 0x09,
+        0x00, 0x00, 0x05 /*zp*/, 0x0D /*abs*/, 0x0F, 0x15 /*zpx*/, 0x1D /*abx*/, 0x1F,
+        0x00, 0x19 /*aby*/, 0x12, 0x00, 0x07, 0x11 /*izy*/, 0x17, 0x01 /*izx*/,
+        0x00, 0x00, 0x00, 0x03, 0x13, 0x09 /*imm*/, 0x00, 0x09,
```

Comment:
Please reformat this section is a suitable way.

Reply:
It was just my transient modification to understand that table when I analyzed the wrong generated SAC/SIR opcodes. By chance I left it on the code during commit. That part has been just reverted. Sorry for the inconvenience.